docs/sovereign/sov-esdt-safe.md

```diff
@@ -0,0 +1,61 @@
+# Sov-ESDT-Safe
+![From Sovereign](../../static/sovereign/from-sovereign.png)
```

Comment:
- Instead of Bridge Service, rename to Sovereign Nodes
- Add the Bridge Service in Main Chain on the right side

Reply:
Also update for Introduction page